src/pages/tutorials/build-an-nft-marketplace/index.md

```diff
@@ -431,543 +443,544 @@ This section guides you step-by-step in setting up an intuitive frontend.
-          </SwipeableViews>
-          <MobileStepper
-            variant="text"
-            steps={Array.from(nftContratTokenMetadataMap!.entries()).length}
-            position="static"
-            activeStep={activeStep}
-            nextButton={
-              <Button
-                size="small"
-                onClick={handleNext}
-                disabled={
-                  activeStep ===
-                  Array.from(nftContratTokenMetadataMap!.entries()).length - 1
+                  >
+                    Next
+                    <KeyboardArrowRight />
+                  </Button>
                 }
-              >
-                Next
-                <KeyboardArrowRight />
-              </Button>
-            }
-            backButton={
-              <Button
-                size="small"
-                onClick={handleBack}
-                disabled={activeStep === 0}
-              >
-                <KeyboardArrowLeft />
-                Back
-              </Button>
-            }
-          />
-        </Box>
-```
+                backButton={
+                  <Button
+                    size="small"
+                    onClick={handleBack}
+                    disabled={activeStep === 0}
+                  >
+                    <KeyboardArrowLeft />
+                    Back
+                  </Button>
+                }
+              />
+            </Box>
+    ```
 
 Finally, your imports at beginning of the file should be like this :
 
-```typescript
-import SwipeableViews from "react-swipeable-views";
-import OpenWithIcon from "@mui/icons-material/OpenWith";
-import {
-  Box,
-  Button,
-  CardHeader,
-  CardMedia,
-  MobileStepper,
-  Stack,
-  SwipeableDrawer,
-  TextField,
-  Toolbar,
-  useMediaQuery,
-} from "@mui/material";
-import Card from "@mui/material/Card";
-import CardContent from "@mui/material/CardContent";
-import {
-  AddCircleOutlined,
-  Close,
-  KeyboardArrowLeft,
-  KeyboardArrowRight,
-} from "@mui/icons-material";
-import Paper from "@mui/material/Paper";
-import Typography from "@mui/material/Typography";
-import { useFormik } from "formik";
-import React, { useEffect, useState } from "react";
-import * as yup from "yup";
-import { TZIP21TokenMetadata, UserContext, UserContextType } from "./App";
-import { useSnackbar } from "notistack";
-import { BigNumber } from "bignumber.js";
-import { address, bytes, nat } from "./type-aliases";
-import { char2Bytes } from "@taquito/utils";
-import { TransactionInvalidBeaconError } from "./TransactionInvalidBeaconError";
-```
-
-and some variables inside your `MintPage` Component function
-
-```typescript
-const [activeStep, setActiveStep] = React.useState(0);
-
-const handleNext = () => {
-  setActiveStep((prevActiveStep) => prevActiveStep + 1);
-};
-
-const handleBack = () => {
-  setActiveStep((prevActiveStep) => prevActiveStep - 1);
-};
-
-const handleStepChange = (step: number) => {
-  setActiveStep(step);
-};
-```
+    ```typescript
```

Comment:
This appears broken in the preview:
<img width="724" alt="Screenshot 2023-10-17 at 3 05 36 PM" src="https://github.com/trilitech/tezos-developer-docs/assets/6494785/36ae8994-79e3-4692-a466-114c986810a6">